utilities/shared_disk.py

```
@@ -253,58 +263,373 @@ def verify_shared_disk_data(
+                _win_read_marker(ctx.ssh_vm1, drive1, test_file_vm2, "Data from VM2", "VM1")
+
+                return True
+        except (
+            SSHException,
+            AuthenticationException,
+            NoValidConnectionsError,
+            ChannelException,
+            GuestCommandError,
+            AssertionError,
+        ) as e:
+            LOGGER.warning(f"Shared disk verification failed: {type(e).__name__}: {e} - retrying...")
```

Ruff (0.15.18)

[warning] Logging statement uses f-string

(G004)